mysqld_exporter.go

```diff
@@ -160,6 +160,16 @@ func customizeTLS(sslCA string, sslCert string, sslKey string) error {
 	return nil
 }
 
+func getdsnByRequest(r *http.Request)string{
+	user := r.URL.Query().Get("user")
+	password := r.URL.Query().Get("pass")
```

Comment:
Passing credentials over URL params is insecure. This is not allowed.

Reply:
thx for reply. How about  write credentials in config , url will carry a parameter key  for point credentials

Reply:
Yes, that's more reasonable. This is how we do things with the snmp_exporter.